### PR TITLE
Removed extraneous requirements and added paver task to install page

### PR DESCRIPTION
**Pull request by Muddasser**

### CODE REVIEW

**Comment by Muddasser** [comment]
@benpatterson @raeeschachar Please review. I have removed extraneous requirements and also added paver task to install edx-platfrom page objects.

File: pavement.py [hunk 1]
```diff
@@ -17,6 +17,23 @@
...
+    repo_root = path(__file__).dirname()
+    # Path to find the git address of repo
+    requirement_path = path(os.path.join(repo_root, 'requirements', 'local.txt'))
+    # Path to store the repo
+    lib_path = path(os.path.join(repo_root, 'lib'))
+    # Path to setup.py of pages package
+    page_obj_setup_path = path(os.path.join(lib_path, 'edx-platform', 'common', 'test', 'acceptance', 'setup.py'))
+    sh("pip install -r {req} --src={lib}".format(req=requirement_path, lib=lib_path))
+    # Install pages
+    sh("python {setup} install".format(setup=page_obj_setup_path))
```
Comment (benpatterson) [inline pavement.py]: Could you add a print statement or something else to the console to explain that we are now installing the page objects?

File: requirements/base.txt [hunk 2]
```diff
@@ -1,255 +1,7 @@
...
-numpy==1.6.2
-oauth2==1.9.0.post1
-oauthlib==0.7.2
-paramiko==1.9.0
--e git+https://github.com/dgrtwo/ParsePy.git@7949b9f754d1445eff8e8f20d0e967b9a6420639#egg=parse_rest-dev
+selenium==2.53.1
 path.py==7.2
-pathtools==0.1.2
+requests==2.0.1
+nose
```
Comment (benpatterson) [inline requirements/base.txt]: Could you pin a specific version?

File: requirements/base.txt [hunk 3]
```diff
@@ -1,255 +1,7 @@
...
-Werkzeug==0.11.5
-wheel==0.24.0
-wrapt==1.10.5
-XBlock==0.4
--e git+https://github.com/open-craft/xblock-poll@e7a6c95c300e95c51e42bfd1eba70489c05a6527#egg=xblock_poll-origin_edx-release
-xmltodict==0.4.1
-Zendesk==1.1.1
-zope.interface==4.1.3
+bok-choy==0.5.2
+fabric==1.8.0
```
Comment (benpatterson) [inline requirements/base.txt]: Should this be removed?

File: requirements/local.txt [hunk 4]
```diff
@@ -1,2 +1,2 @@
 # edx-platform page objects
--e git+https://github.com/edx/edx-platform.git@1bbfbedb1f4b1c8bb65f6c9249cdcf2bc308bf49#egg=edx-platform
+-e git+https://github.com/edx/edx-platform.git@9cb660a7d288bad618ba9a12126a79f63028bf97#egg=edx-platform
```
Comment (benpatterson) [inline requirements/local.txt]: Can we pull directly off of master instead of a specific hash?

File: requirements/base.txt (hunk 3 above)
Comment (raeeschachar) [inline requirements/base.txt]: Yup, removed.

File: requirements/base.txt (hunk 2 above)
Comment (raeeschachar) [inline requirements/base.txt]: Done.

File: pavement.py (hunk 1 above)
Comment (raeeschachar) [inline pavement.py]: Done.

File: requirements/local.txt (hunk 4 above)
Comment (raeeschachar) [inline requirements/local.txt]: Would be better. Done.

**Comment by raeeschachar** [comment]
@benpatterson I merged this PR with mine, comments addressed on https://github.com/edx/edx-e2e-tests/pull/54/commits/0595d1e7e682b364b0d39c9df7a4083ee44bfb2d

**Comment by benpatterson** [comment]
Cool.

**Pull request merged**